library/helpers/attackPath.ts

```diff
@@ -1,3 +1,9 @@
+import { isPlainObject } from "./isPlainObject";
+import { tryDecodeAsJWT } from "./tryDecodeAsJWT";
+
+// Maximum match count to return
+const MAX_MATCH_COUNT = 10;
```

Comment:
```suggestion
const MAX_MATCH_COUNT = 1;
```

Since we'll only report one match?